stdlib/public/core/StringObject.swift

```diff
@@ -228,6 +228,22 @@ extension _StringObject {
       countAndFlags: countAndFlags)
   }
 
+#if arch(arm64)
+  // Initializer to use for constant ObjC tagged values
+  @inline(__always)
+  internal init(
+    objcConstantTaggedPointerBits: UInt64, countAndFlags: CountAndFlags
```

Comment:
"objc", "constant", and even "tagged" are not really the relevant concepts here, and the name could also include small packed NSStrings. Also, those aren't the `pointerBits` like in the prior initializer if they already include the discriminator.